src/Microsoft.Extensions.Configuration.Abstractions/ConfigurationPath.cs

```diff
@@ -11,26 +11,44 @@ namespace Microsoft.Extensions.Configuration
     /// </summary>
     public static class ConfigurationPath
     {
+        /// <summary>
+        /// The deliminiter used for keys is ":".
+        /// </summary>
         public static readonly string KeyDelimiter = ":";
 
-        public static string Combine(params string[] pathSegements)
+        /// <summary>
+        /// Combines all several path segments into one path.
```

Comment:
`all several` --> perhaps just remove both words. The sentence will retain the same meaning.